staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -249,14 +268,32 @@ func (o *AuditWebhookOptions) AddFlags(fs *pflag.FlagSet) {
 		"Strategy for sending audit events. Blocking indicates sending events should block"+
 			" server responses. Batch causes the webhook to buffer and send events"+
 			" asynchronously. Known modes are "+strings.Join(pluginwebhook.AllowedModes, ",")+".")
+	fs.IntVar(&o.BatchConfig.BufferSize, "audit-webhook-batch-buffer-size",
```

Comment:
Given that we're adding a more general buffering mechanism in 1.10, should these flags be more generic? Or do we still want the option for setting different batch params for the log and webhook backends?

Reply:
I had in mind making it configurable separately & creating those flags automatically with a given prefix (`audit-webhook-` or `audit-log-`)